packages/smart-wallet/src/smartWallet.js

```diff
@@ -433,7 +488,9 @@ export const prepareSmartWallet = (baggage, shared) => {
               helper.updateBalance(purse, balance);
             },
             fail(reason) {
-              console.error(address, `failed updateState observer`, reason);
+              console.error(
+                `*** ${address} failed updateState observer, ${reason} ***`,
```

Comment:
why the stars?

Reply:
I think I was making it more visible. I'll replace it with '🚨'.